core/src/v2/actix.rs

```diff
@@ -316,11 +316,20 @@ macro_rules! impl_param_extractor ({ $ty:ty => $container:ident } => {
                 op.parameters.push(Either::Right(Parameter {
                     in_: ParameterIn::$container,
                     required: def.required.contains(&k),
-                    name: k,
                     data_type: v.data_type,
                     format: v.format,
                     enum_: v.enum_,
                     description: v.description,
+                    items: v.items.as_ref().map(|schema| {
```

Comment:
Nested parameters can go beyond one level in schema, so it's better if we probably leave it at `items: v.items.clone()`?

Reply:
Here `v.items` is `Option<DefaultSchemaRaw>` rather than `Option<Items>` so it needs to be mapped.
I thought that this part of the code was only being used to produce items for query / path parameters in which case nested items would only work with different values for `collection_format` so that will need to be specified somewhere.
I think we need `collection_format` before this can be merged.

Reply:
Now that I have read the documentation of serde_qs a bit better I see what you mean, nesting in queries is not something I have used.
The openapi v2 spec doesn't allow for objects in query parameters so the definitions generated for serde_qs will need to create a separate parameter for each nested field. I will have a try at this `serde_qs`-specific generation